app/src/main/java/org/greenstand/android/TreeTracker/view/Text.kt

```diff
@@ -24,10 +28,12 @@ object TextStyles {
 
 @Composable
 fun BoxScope.TopBarTitle() {
-    Text(
-        "TREETRACKER",
-        modifier = Modifier.align(Alignment.Center),
```

Comment:
We want to keep the center alignment. You'll noticed the icon is off center when you look at it.